charts/mod-authtoken/values.yaml

```diff
@@ -109,6 +109,7 @@ extraEnv: |
 dbConnectParameters: db-connect-modules 
 # kafkaConnectParameters: db-connect-modules
 # awsConnectParameters: s3-credentials
+rtrEnabled: true
```

Comment:
As in https://github.com/folio-org/pipelines-shared-library/pull/423/files#diff-f010ef75264e055603df0926f9b143f11c1c7adaa55604abbb5c11a58d8b42c1
We set rtrEnabled to true, than better to set here false as default value